dotcom-rendering/src/web/bork/fcp.ts

```diff
@@ -0,0 +1,19 @@
+export const fcp = (): void => {
+	try {
+		const hashKey = 'bork-fcp-';
+		const hash = window.location.hash.substring(1);
+		if (
+			hash.startsWith(hashKey) &&
+			CSS.supports('animation-duration', 'var(--fake-var)')
+		) {
+			const root = document.documentElement;
+			root.style.setProperty(
+				'--bork-fcp-amount',
+				hash.replace(hashKey, '') + 'ms',
```

Comment:
Should this be an additional delay once loaded, or an absolute time from the beginning of page load?